content/en/docs/examples/bookinfo-multicluster/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Bookinfo Application - Multicluster
+description: Deploys a sample application across a multicluster mesh.
+weight: 11
+keywords: [multicluster]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+This examples complements the [simplified multicluster setup procedure](/docs/setup/install/multicluster/simplified).
+It provides a simple procedure to deploy Istio's classic [Bookinfo](/docs/examples/bookinfo) sample application across
+a multicluster mesh.
+
+## Getting it running
+
+1. Start by following [these instructions](/docs/setup/install/multicluster/simplified) which will show you how to
+configure a 3 cluster mesh.
+
+1. In the working directory created during the above step, download the `setup-bookinfo.sh` script:
+
+    {{< text bash >}}
+    $ curl @samples/multicluster/setup-bookinfo.sh@ >./setup-bookinfo.sh
```

Comment:
This is problematic because it shows up as `curl samples/multicluster/setup-mesh.sh` in the doc. It may be a link to the right place, but the copy / paste flow is busted.

Reply:
Fixed. Good catch.